src/org/opensolaris/opengrok/web/api/RestApp.java

```diff
@@ -30,7 +30,7 @@
 public class RestApp extends ResourceConfig {
 
     public RestApp() {
-        packages("org.opensolaris.opengrok.web.api.controller", "org.opensolaris.opengrok.web.api.filter");
+        packages("org.opensolaris.opengrok.web.api.v1.controller", "org.opensolaris.opengrok.web.api.filter");
```

Comment:
`api.v1.filter` also?

Reply:
Why? I thought that this filter will probably be used later as well. Until we come up with different security approach. 

Either way, I think that it won't be a problem to rename it.

Reply:
I didn't study the framework but this creates the app looking for controllers in v1.controller and it should be looking into v1.filter for filters to remain consistent.

If anyone wants to create other filter in other version there would be v2 and one can extend v1 filters or create new ones.